server/konnectors/linkedin.coffee

```diff
@@ -0,0 +1,328 @@
+
+# Vendor
```

Comment:
Can you remove the vendor comments. It's a convention to write requirements first. So comments are useless here.